build/StaticLibCheck.cs

```diff
@@ -0,0 +1,207 @@
+using Cake.Common.Tools.VSWhere.Latest;
+namespace BuildScripts;
+
+public class StaticLibCheckTask
```

Comment:
A comment here stating the purpose of this class and how it works will help future development.

Reply:
Also `StaticLibCheckTask` -> `StaticLibCheck` for the name.